src/ufs.rs

```diff
@@ -359,7 +359,7 @@ impl Filesystem for Ufs {
 	fn lookup(&mut self, _req: &Request<'_>, pino: u64, name: &OsStr, reply: fuser::ReplyEntry) {
 		let pino = transino(pino);
 
-		let f = || {
+		let mut f = || {
```

Comment:
Why does this need to be mutable?

Reply:
`f: impl FnMut` => `f` has to mut `mut` to be called.
If `f: impl Fn`, which it doesn't, then you could drop the `mut`.
See [FnMut](https://doc.rust-lang.org/std/ops/trait.FnMut.html)

Reply:
But why does it need to be FnMut?  And where does `self` come into play?

Reply:
The whole reason I do this closure + run, is that I want to skip proper error handling for now.
Once the driver is good enough, I'll go through every syscall and do proper error handling.

Reply:
In this case I'd have to replace the two `self.read_inode(...)?` with `match self.read_inode(...) { ... }`.

Reply:
What do you have in mind for "proper error handling", anyway?  I don't think you can do much better than log::debug! .

Reply:
I meant that I don't want to worry about error handling now, and do it later, once more important things are done,
like extended attributes.

Reply:
I meant that I don't think you'll ever be able to "handle" errors in any way other by logging them and passing them back to the kernel.

Reply:
ok